core/ordering/cosipbft/pbft/mod.go

```diff
@@ -831,5 +849,9 @@ func (obs observer) NotifyCallback(event interface{}) {
 // be found with n = 3*f+1 where n is the number of participants.
 func calculateThreshold(n int) int {
 	f := (n - 1) / 3
+	if f == 0 {
+		return n
+	}
+
 	return 2 * f
```

Comment:
```suggestion
	if n <= 3 {
		return n
	}
	f := (n - 1) / 3

	return 2 * f
```

But it's the same thing...